docs-markdown/markdownlint-custom-rules/strings.js

```diff
@@ -12,9 +12,11 @@ module.exports.monikerSyntax = `Bad syntax for moniker. Only "moniker range" is
 module.exports.noLocSyntax = `Bad syntax for non-localization.`;
 module.exports.noLocNoDash = `Bad syntax for non-localization. "no-loc" requires a dash.`;
 module.exports.noLocMissingTextAttribute = `Bad syntax for non-localization. Missing the "text" attribute.`;
+module.exports.NoLocNonAllowedAttribute = `Bad syntax for non-localization. "___" is not an allowed attribute.`;
 module.exports.noLocColonsIncorrect = `Bad syntax for non-localization. Make sure "no-loc" begins and ends with ":::"`;
 module.exports.noLocCaseSensitive = `Bad syntax for non-localization. ""no-loc" and "text" must be lower-case.`;
 module.exports.noLocNoQuotes = `Bad syntax for non-localization. Quotes required.`;
+module.exports.noLocDoubleQuotes = `Bad syntax for non-localization. Double Quotes (") required.`;
```

Comment:
@bharney0 Super small.  **Quotes** can probably be lowercase but other than that everything works great.